docs/install_lalsuite.rst

```diff
@@ -137,10 +145,17 @@ If you are running a pipeline that uses the old LALApps programs ``lalapps_inspi
 
 .. note::
 
-    The LALApps build above builds static binaries, so you will need static libraries for fftw, glibc, etc. installed on your system to do this. These libraries are present by default in a LIGO Data Grid environment. If you do not wish to build static LALApps programs, the omit the ``--enable-static-binaries`` option to the configure script.
+    The LALApps build above builds static binaries, so you will need static libraries for fftw, glibc, etc. installed on your system to do this. These libraries are present by default in a LIGO Data Grid environment. 
 
+.. note::
+
+    Installing LALApps binaries in ``$VIRTUAL_ENV/bin`` as above may clash with executables previously installed in ``$VIRTUAL_ENV/opt/lalsuite/bin``, the destination used by the standard lalsuite build.  If in doubt, run ``which lalapps_inspinj``` to determine which executable is in your path.
 
-In addition to lalsuite, the generation of certain template waveforms (e.g.  the reduced order model implementations of SEOBNRv2 and SEOBNRv4) requires addition data files from the `lalsuite-extra repository <https://svn.ligo.caltech.edu/svn/lalsuite-extra/>`_. These data can either be obtained by downloading and installing lalsuite-extra into your virtual environment or using a copy of the data from the CERN virtual filesystem.
+=========================================
+Additional data files from lalsuite-extra
+=========================================
+
+In addition to lalsuite, the generation of certain template waveforms (e.g. the reduced order model implementations of SEOBNRv2 and SEOBNRv4) requires data files from the `lalsuite-extra repository <https://svn.ligo.caltech.edu/svn/lalsuite-extra/>`_. These data can either be obtained by downloading and installing lalsuite-extra into your virtual environment or using a copy of the data from the CERN virtual filesystem.
```

Comment:
We need to update the extra repository location. I'll propose that change via pull request.

Reply:
This should now be https://git.ligo.org/lscsoft/lalsuite-extra and the repository clone command be:

git clone https://git.ligo.org/lscsoft/lalsuite-extra.git

Reply:
Are you going to make this change yourself or are you now asking for it to be in this pull?

Reply:
I want it in this pull. I can make it directly myself if you say "yes, that's okay"

Reply:
"Yes, that's okay" 
(just want to see what's going to happen)